style.css

```diff
@@ -85,14 +86,41 @@ li {
   font-weight: 500;
 }
 
-#nav_menu {
+.hidden {
   display: none;
 }
 
-/* i added this to remove some white space that appeared
-after i put the burger menu inside a navbar */
+.overflow {
+  overflow: hidden;
+}
+
+.toggled-menu {
+  background: #fff;
+  display: block;
+  position: absolute;
+  top: -24px;
+  left: 0;
+  height: 100%;
+  padding-top: 100px;
+  padding-left: 24px;
+  z-index: 0;
+  border-radius: 0 0 48px 48px;
+}
+
+.nav-link {
+  color: #3a4a42;
+  display: block;
+  font-size: 32px;
+  font-weight: 600;
+  margin-bottom: 40px;
+}
```

Comment:
- [ ] As requested by the previous reviewer, kindly add a line height of 44px to match the gap between the nav-links in the template `line-height: 44px;` it's really important to pay attention to detail.